src/modules/tas_studio/mod.rs

```diff
@@ -1870,6 +1887,13 @@ fn add_frame_bulk_hud_lines(text: &mut Vec<u8>, bulk: &FrameBulk) {
                 StrafeType::ConstYawspeed(yawspeed) => {
                     write!(text, "turn rate: {yawspeed:.0}").unwrap();
                 }
+                StrafeType::MaxAccelYawOffset(start, target, accel) => {
+                    // The values are so tiny that only this would make it sensible.
```

Comment:
This is better than just showing them with 2 digits? It could be confusing I guess

Reply:
Here is a sample part where I use it.

```
s51-------|------|------|0.010000001|- -5.4011025 -5.4011025 0|-|43
s51-------|------|------|0.010000001|- -5.635861 -5.635861 0|-|268
s50-j-----|------|------|0.010000001|- -0.70690674 -0.70690674 0|-|134
s50-j-----|------|------|0.010000001|- -2.2149327 -2.597555 0|-|19
s51-j-----|------|------|0.010000001|- -0.59532917 -0.59532917 0|-|169
s50-j-----|------|------|0.010000001|- -10.585796 -1.0256441 0.12860943|-|982
s01-j-----|------|------|0.010000001|-|-|221
s50-j-----|------|------|0.010000001|- -9.952005 -0.9520051 0.1|-|941
s50-j-----|------|------|0.010000001|- -0.13668747 -0.13668747 0|-|199
s51-j-----|------|------|0.010000001|- 0.32988316 -0.29589385 0|-|295
s51-j-----|------|------|0.010000001|- 2.9388967 4.460467 0|-|7
s51-j-----|------|------|0.010000001|- 2.9388967 4.460467 0|-|72
```

I think 2 digits (times 10) might not be descriptive enough. And also we have space for 3.